### DISCUSSION TITLE
Etherpad errors log file

### DISCUSSION THREAD

**Discussion by soberrhino** [ether/etherpad-lite#4734] [Q&A]
Hi,
Silly question but I could not find the information in documentation. Where should I find etherpad errors log file?
Thanks,
Yves

**Comment by JohnMcLear** [comment]
Do you know how to see your log file?

**Reply by soberrhino** [reply]
Actually, no, I'm afraid.

**Reply by JohnMcLear** [reply]
How did you start etherpad?

**Comment by soberrhino** [comment]
I installed etherpad on a mutualized server by downloading 1.8.3, adding a few symbolic links (sorry guide is in French <https://faq.o2switch.fr/nodejs/comment-installer-etherpad-hebergement-o2switch>) and configuring Setup Node JS App.
It works quite well but sometimes the server hangs. I suspect etherpad but would like to have a look at some logs.

**Reply by JohnMcLear** [reply]
Please use 1.8.7.  what operating system does the guide cover?  Sorry but I can't visit the link
